tests/test_torchinductor.py

```diff
@@ -165,6 +169,84 @@ class SweepInputsCpuTest(SweepInputs2, TestCase):
 SweepInputsCpuTest.populate()
 
 
+class TestIndexingSimplification(unittest.TestCase):
+    def test_indexing_simplification(self):
```

Comment:
tests are pretty cool and show how simplifications are supposed to work, but (if we are to use them as documentation) would benefit from brief comments on why those results are expected)

Reply:
fixed